grpcgcp/gcp_balancer.go

```diff
@@ -370,6 +375,13 @@ func (gb *gcpBalancer) getReadySubConnRef(boundKey string) (*subConnRef, bool) {
 	return nil, false
 }
 
+func (gb *gcpBalancer) getSubConnRoundRobin() *subConnRef {
```

Comment:
Do you want to check for watermark num connections? I guess that complicates the code.

Reply:
No, with this option on, we want to always land a binding call in a round-robin manner regardless of channel health or exceeding the watermark limit. This will be used for the cases where affinity keys are created beforehand in large quantity so we can wait until a channel recovers or a call waits in the queue.